test/common/crudtester.go

```diff
@@ -224,7 +224,9 @@ func (c *FederatedTypeCrudTester) CheckPlacementChange(template, placement, over
 		if err != nil {
 			c.tl.Fatalf("Error retrieving cluster names: %v", err)
 		}
-		clusterNames = c.deleteOneNonPrimaryCluster(clusterNames)
+		// Removing the resource from an arbitrary cluster (even the
+		// primary one) is safe for any resource but namespaces.
```

Comment:
Seems like we need a check for namespaces here.

Also, I'm not understanding how this commit message relates to these code changes. Am I missing something?